docs/tutorial/managing-ontology-project.md

```diff
@@ -0,0 +1,54 @@
+
+## What do we need? The Project Management Toolbox
+
+Every ontology or group of related ontologies (sometimes it is easier to manage multiple ontologies at once, because their scope or technical workflows are quite uniform or they are heavily interrelated) should have:
+
+1. *at least two teams*, Curation Team and Technical Team, with clearly defined members. We recommend to create two teams on GitHub and keep its members always up to date (i.e. remove members that are not actively participating). Many of our projects furthermore have a third "core team", which is a more liberal team containing everyone from stakeholders, principal investigators, curators and users.
+2. *two distinct project boards*: One for the Curation Team, and one for the Technical Team. The details on how to design the boards is up to the respective teams, but we found a simple 4 stage board with sections for `To Do` (issues that are important but not urgent), `Priority` (issues that are important and urgent), `In Progress` (issues that are being worked on), `Under review` (issues that need review) and `Done` (finished issues) the most useful. The least useful column on a project board is `Backlog`, as it clutters the board and creates a massive overhead in the rare times the team meets to discuss the next steps.
+3. *A documentation system* (often realised using `mkdocs` in OBO projects) with a page listing the members of the team. This page should provide links to all related team pages from Github and their project boards, as well as a table listing all current team members with the following information:
+    - Name
+    - ORCiD
+    - Funding Information
+    - Allocated FTEs (0 if on volunteering basis)
+    - Associated teams
+    - Role 
+    - Responsibilities (What kind of issues can they be assigned to review? How are they involved in the Project?)
+
+## Responsibilities
+
+1. Effective Ontology Pipeline Developers (OPDs) are extremely rare and are typically active across many different projects. Therefore *their attention is scattered*. Understanding and accepting this is key for the following points.
+1. Principal Investigators *explicitly assign target weekly hours* for Ontology Curators and Ontology Pipeline Developers to the project. These should be captured on the documentation systems team page (see above).
+2. The Ontology Curators are responsible for the entire Curation Team Board and the `To Do` and `Priority` columns of the Technical Team. The later is important: it is the job of the curation team to _prioritise the technical issues_. The Technical Team can add tickets to the `To Do` and `Priority` columns, but this usually happens only in response to a request from the Curation Team.
+3. When the technical team meets, the Principal Ontology Curator(s) (POC) are present, i.e. *the POCs are members of the technical team as well*. They will help clarifying the `Priority` tickets. The Technical Team is responsible to 
```

Comment:
Might want to avoid POC acronym  - danger of confusion with 'People Of Color' - common in US.

Reply:
If we use Editor instead of Curator, this can be POE.

Reply:
Done, thank you!